app/grants/templates/grants/cart/eth.html

```diff
@@ -49,7 +49,7 @@
           <div v-else-if="grant.is_on_team" class="clr-match-box font-caption font-weight-semibold d-flex flex-nowrap align-items-center">
             Cannot match own grant
           </div>
-          <div v-else-if="grant.is_clr_eligible == true" class="clr-match-box d-flex flex-nowrap align-items-center">
+          <div v-else-if="grant.is_clr_eligible == true && grant.opt_out_clr == false" class="clr-match-box d-flex flex-nowrap align-items-center">
```

Comment:
can we create a new property is_grant_eligible_for_clr which combines `grant.is_clr_eligible == true && grant.opt_out_clr == false` and we just use that ?

Reply:
Totally agree with this since there's so much repetition. Also, I'd recommend sticking to use either 
`grant.opt_out_clr == false` or `!grant.opt_out_clr` but not both.

Reply:
here

```
    @property
    def is_grant_eligible_for_clr(self):
        return self.is_clr_eligible == True and self.opt_out_clr == False
 ```

is there a need to keep both `is_clr_eligible` and `is_grant_eligible_for_clr` ?